src/lib/holocene/copyable/button.svelte

```diff
@@ -29,6 +29,6 @@
 
 <style lang="postcss">
   .copy-button {
-    @apply m-1 rounded-md border-2 border-[transparent] hover:border-indigo-600 hover:bg-gradient-to-br hover:from-blue-100 hover:to-purple-100 hover:shadow-focus hover:shadow-blue-600/50 focus-visible:border-indigo-600 focus-visible:bg-gradient-to-br focus-visible:from-blue-100 focus-visible:to-purple-100 focus-visible:shadow-focus focus-visible:shadow-blue-600/50 focus-visible:outline-none;
+    @apply m-1 rounded-md border-2 border-[transparent] bg-transparent text-white hover:bg-slate-800 focus-visible:border-indigo-600 focus-visible:bg-slate-800 focus-visible:outline-none;
```

Comment:
We'll need styling for light mode too. This isn't only used within a `CodeBlock` component, it's also used in the `Copyable` component which can be on any background (e.g. with the `Workflow ID` ⬇️ ).

<img width="336" alt="Screenshot 2024-04-04 at 6 04 29 PM" src="https://github.com/temporalio/ui/assets/15069288/454f0392-6e3d-4489-ab67-47c9858d01eb">

Reply:
@laurakwhit would it make sense to have a conditional for when it's used in the `CodeBlock`? That way we can have it show the darkmode styles in the `CodeBlock` component?

Reply:
It looks like [we're currently passing](https://github.com/temporalio/ui/blob/main/src/lib/holocene/code-block.svelte#L168) a `class` that overrides the styles. We might just want to do something similar, but I think a prop wouldn't be a bad idea either.

Reply:
@laurakwhit So I tried to use the class, and the other styles are overriding the things I'm passing in. Now sure why? What do you think?